### PR TITLE
Transfer history

### PR DESCRIPTION
**Pull request by hello-brsd**
#### Steps to reproduce:
go to account page

#### Trello card / screenshot / wireframe link:
https://trello.com/c/uk1I2HhG/207-refactor-transaction-history-as-new-wireframe

#### Is connection necessary to test? If so which network?
- [x] local RPC
- [x] Rinkeby
- [x] Main Ethereum Network

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 6e953c03e069cd100d7e656a8efb87686deb9c7e

https://app.netlify.com/sites/augmint/deploys/5b4da4f2b3127475f8e841ee

File: src/components/augmint-ui/button/styles.js [hunk 1]
```diff
@@ -22,24 +22,37 @@ const BaseButton = `
...
+        background-color: ${theme.colors.secondaryDark};
+      }
+    }
+    
+    &.primary {
+        padding: 8px 30px;
+        background-color: ${theme.colors.primary};
         color: ${theme.colors.white};
-        box-shadow: 0 2px 14px rgba(0, 0, 0, 0.5);
+        font-size: 16px;
```
Comment (gyurka00) [inline src/components/augmint-ui/button/styles.js]: please use remCalc functin from styles/theme.js to set font-size

`import { remCalc } from "styles/theme";
...
font-size: ${remCalc(16)`

File: src/components/transaction/index.js [hunk 2]
```diff
@@ -0,0 +1,106 @@
+import React from "react";
+import styled from "styled-components";
+import theme from "styles/theme";
+import AccountAddress from "components/accountAddress";
+import HashURL from "components/hash";
+import { connect } from "react-redux";
+
+export const TxDate = styled.span`
+    font-size: 12px;
```
Comment (gyurka00) [inline src/components/transaction/index.js]: the same problem
please use remCalc function

File: src/components/transaction/index.js [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+`;
+
+export const TxTitle = styled.span`
+    font-weight: bold;
+`;
+
+export const TxDetails = styled.span`
+    font-size: 12px;
```
Comment (gyurka00) [inline src/components/transaction/index.js]: the same problem
please use remCalc function

File: src/containers/account/components/TransferList.js [hunk 4]
```diff
@@ -1,88 +1,117 @@
...
-                            <Row columns={1}>
-                                <Col>{tx.args.narrative}</Col>
-                            </Row>
-                        )}
-                    </MyGridTable>
-                </MyListGroup.Row>
-            ));
+            transfers.reduce((balance, tx, index, all) => {
+                const amount = index > 0 ? all[index - 1].signedAmount : 0;
+                return (tx.balance = Math.round(balance * 100 - amount * 100) / 100);
```
Comment (gyurka00) [inline src/containers/account/components/TransferList.js]: All the time enough 2 digits?
(We can use  bignumber.js for the  float number calculation.)

Comment (szerintedmi) [reply src/containers/account/components/TransferList.js]: as there is only additions I think we don't need bignumbers here , rounding is enough (and A-EUR balances will stay in normal ranges unlike wei values)

Comment (gyurka00) [reply src/containers/account/components/TransferList.js]: ok

**Pull request merged**